scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+| external_sample_names | Strings describing the unique sample IDs in the Terra data model. | Array of strings |
+| input_vcfs | Cloud paths to the sample gVCF files. | Array of files |
+| input_vcf_indexes | Cloud paths to the sample gVCF index files. | Array of files |
+| callset_identifier | String used as the name of the filter model and as the prefix to the names of the BigQuery extract tables and final joint VCF shards. | String |
+
+## Tasks and tools
+
+The [GVS workflow](https://github.com/broadinstitute/gatk/blob/rc-vs-483-beta-user-wdl/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) imports additional WDL scripts that contain the workflow tasks. Links to these additional WDL scripts (subtasks/subworkflows) are provided below.
+
+Overall, the GVS workflow:
+
+1. Imports sample genomes.
+1. Trains the filtering model.
+1. Extracts VCF files.
+
+To see specific tool parameters used in each step of the workflow, select the WDL link in the table; then find the task and view the `command {}` section of the task in the WDL script.
+
+To view or use the exact tool software, see the task's Docker image, which is specified in the task `runtime {}` section as `docker: `.
+
+### 1. Import sample genomes
+
+To import sample genomes, the GVS workflow calls two subworkflows detailed below.
+
+#### A. GvsAssignIds
+
+The [GvsAssignIds subworkflow (alias = AssignIds)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsAssignIds.wdl) assigns GVS-specific IDs to each of the samples in the sample set which are used as an access pattern. The IDs enable easy access to specific sample data. GvsAssignIds imports an additional workflow, [GvsCreateTables](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsCreateTables.wdl).
+
+#### B. GvsImportGenomes
+
+The [GvsImportGenomes subworkflow (alias = ImportGenomes)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsImportGenomes.wdl) validates that sample gVCF files contain required annotations and loads samples into several tables in BigQuery. GvsImportGenomes imports an additional workflow, [GvsUtils (alias = Utils)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsUtils.wdl).
+
+### 2. Train the filtering model
+
+To train the filtering model, the GVS workflow calls two subworkflows detailed below.
+
+#### A. GvsCreateAltAllele
+
+The [GvsCreateAltAllele subworkflow (alias = CreateAltAllele)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsCreateAltAllele.wdl) splits alternate alleles and calculates additional annotations to be used for filtering. GvsCreateAltAllele imports an additional workflow, [GvsUtils (alias = Utils)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsUtils.wdl).
+
+#### B. GvsCreateFilterSet
+
```

Comment:
What is the output of this task? The next section talks about VCFs, but we've been working with GVCF up till this point.

Reply:
another table in the dataset -- not something the user needs to understand, but it's the output of this that we use to create the filter

Reply:
this is the first part of the BQ only part of the workflow. We are taking data from one table (or several, if the number of samples is >4k) and moving it into another table (partially for data access patterns!!!)

Reply:
I simplified this section significantly so I don't think your question is answered in the doc @ekiernan. However, I hope by simplifying it, it makes it seem less like something users need to know the inner workings of?